op.c

```diff
@@ -8926,28 +8957,35 @@ Perl_utilize(pTHX_ int aver, I32 floor, OP *version, OP *idop, OP *arg)
         use_version = sv_2mortal(new_version(use_version));
         S_enable_feature_bundle(aTHX_ use_version);
 
+        U16 shortver = S_extract_shortver(aTHX_ use_version);
+
         /* If a version >= 5.11.0 is requested, strictures are on by default! */
-        if (vcmp(use_version,
-                 sv_2mortal(upg_version(newSVnv(5.011000), FALSE))) >= 0) {
+        if (shortver >= SHORTVER(5, 11)) {
             if (!(PL_hints & HINT_EXPLICIT_STRICT_REFS))
                 PL_hints |= HINT_STRICT_REFS;
             if (!(PL_hints & HINT_EXPLICIT_STRICT_SUBS))
                 PL_hints |= HINT_STRICT_SUBS;
             if (!(PL_hints & HINT_EXPLICIT_STRICT_VARS))
                 PL_hints |= HINT_STRICT_VARS;
 
-            if (vcmp(use_version, sv_2mortal(upg_version(newSVpvs("5.035000"), FALSE))) >= 0)
+            if (shortver >= SHORTVER(5, 35))
                 free_and_set_cop_warnings(&PL_compiling, pWARN_ALL);
         }
         /* otherwise they are off */
         else {
+            if(PL_prevailing_version >= SHORTVER(5, 11))
+                deprecate_disappears_in("5.40",
+                    "Downgrading a use VERSION declaration to below v5.11 is deprecated");
```

Comment:
The "disappears" wording seems a bit odd here, I would expect "will be fatal in", but the `deprecate_fatal_in()` wording isn't quite right either. Seeing as those are `PERL_CORE` only, and only `deprecate()` is actually used, feel free to reword them.

Reply:
That's fair. I'll have a think on some better wording.